fuzz/README.md

```diff
@@ -104,12 +104,14 @@ FUZZ_FNAME: hfuzz_workspace/hashes_sha256/SIGABRT.PC.7ffff7c8abc7.STACK.18826d9b
 fff400610004
 ```
 
+If you don't see the summary because you fuzzed via `cargo`, run
+
+    checkReport <target>
+
```

Comment:
In 3ea836c9c0615fbcf2beb48d64bc7c52518af452:

We should reword the original text to asy "If fuzz.sh or cycle.sh fails, it will exit..." and there here say "if you can `cargo hfuzz` manually, use `checkReport <target>` to see the report".

Reply:
Also, without manually running `source fuzz-util.sh` this won't work, and once you do, it'll kill your shell. I'll leave it to you whether to update it and update the README, or whether you just want to change the text to use `xxd` directly.